sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -260,6 +262,54 @@ def to_runner_api(self, unused_context):
         default=beam_runner_api_pb2.Trigger.Default())
 
 
+class AfterProcessingTime(TriggerFn):
+  """Fire exactly once after a specified delay from processing time.
+
+  AfterProcessingTime is experimental. No backwards compatibility guarantees.
+  """
+
+  def __init__(self, delay=0):
+    self.delay = delay
+
+  def __repr__(self):
+    return 'AfterProcessingTime(delay=%d)' % self.delay
+
+  def on_element(self, element, window, context):
+    context.set_timer('', TimeDomain.REAL_TIME, self.delay)
+
+  def on_merge(self, to_be_merged, merge_result, context):
+    for window in to_be_merged:
+      if window.end != merge_result.end:
+        context.clear_timer('', TimeDomain.REAL_TIME)
+
+  def should_fire(self, watermark, window, context):
+    return False
```

Comment:
<!--new_thread; commit:a769afcabd2a4b8ad94cae1f1c62e73823e42d7b; resolved:0-->
Can you comment on why this is false?  It is not clear from this code when this trigger actually fires.